tabs/system-setup/9-auto-mount.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh -e
+
+. ../common-script.sh
+
+# Function to display available drives and allow the user to select one
+select_drive() {
+    clear
+    printf "%b\n" "Available drives and partitions:"
+    lsblk -o NAME,SIZE,FSTYPE,MOUNTPOINT,LABEL,UUID | grep -v 'loop' # list all non-loop devices
+    printf "\n"
+    printf "%b\n" "Enter the drive/partition name (e.g., sda1, sdb1): "
+    read -r drive_name
+    # Check if the input is valid
+    if lsblk | grep -q "${drive_name}"; then
+        partition="/dev/${drive_name}"
+    else
+        printf "%b\n" "Invalid drive/partition name!"
+        exit 1
+    fi
+}
+
+# Function to get UUID and FSTYPE of the selected drive
+get_uuid_fstype() {
+    UUID=$(blkid -s UUID -o value "${partition}")
+    FSTYPE=$(lsblk -no FSTYPE "${partition}")
+    NAME=$(lsblk -no NAME "${partition}")
+
+    if [ -z "$UUID" ]; then
+        printf "%b\n" "Failed to retrieve the UUID. Exiting."
+        exit 1
+    fi
+
+    if [ -z "$FSTYPE" ]; then
+        printf "%b\n" "Failed to retrieve the filesystem type. Exiting."
+        exit 1
+    fi
+}
+
+# Function to create a mount point
+create_mount_point() {
+    printf "%b\n" "Enter the mount point path (e.g., /mnt/hdd): "
+    read -r mount_point
+    if [ ! -d "$mount_point" ]; then
+        printf "%b\n" "Mount point doesn't exist. Creating it..."
+        $ESCALATION_TOOL mkdir -p "$mount_point"
+    else
+        printf "%b\n" "Mount point already exists."
+    fi
+}
+
+# Function to update /etc/fstab with a comment on the first line and the actual entry on the second line
+update_fstab() {
+    printf "%b\n" "Adding entry to /etc/fstab..."
+    $ESCALATION_TOOL  cp /etc/fstab /etc/fstab.bak # Backup fstab
+
+    # Prepare the comment and the fstab entry
+    comment="# Mount for /dev/$NAME"
+    fstab_entry="UUID=$UUID $mount_point $FSTYPE defaults 0 2"
+
+    # Append the comment and the entry to /etc/fstab
+    printf "%b\n" "$comment" | $ESCALATION_TOOL  tee -a /etc/fstab > /dev/null
+    printf "%b\n" "$fstab_entry" | $ESCALATION_TOOL  tee -a /etc/fstab > /dev/null
+    printf "%b\n" "" | $ESCALATION_TOOL tee -a /etc/fstab > /dev/null
```

Comment:
```suggestion
    printf "%b\n" "" | "$ESCALATION_TOOL" tee -a /etc/fstab > /dev/null
```
Every $ESCALATION_TOOL needs quoted.